pkg/instance/instance.go

```diff
@@ -27,11 +27,17 @@ import (
 	"github.com/google/syzkaller/vm"
 )
 
+type BuilderTester interface {
+	BuildSyzkaller(string, string) error
+	BuildKernel(string, string, string, string, []byte) (string, error)
+	Test(numVMs int, reproSyz, reproOpts, reproC []byte) ([]error, error)
+}
+
 type Env struct {
```

Comment:
If BuilderTester defines all Env methods, then it would be reasonable to unexport Env type entirely. Nobody should create/use it now.